### PR TITLE
ETH - Fix cache logic of get_all_balance

### PR DESCRIPTION
**Pull request by huazhouwang**
## What does this implement/fix? Explain your changes.
1. 修改缓存逻辑，仅缓存10s

## Does this close any currently open issues?  
If it fixes a bug or resolves a feature request, be sure to link to that issue.
https://github.com/OneKeyHQ/TaskHub/issues/779

## Pull request type
 
_Put an `x` in the boxes that apply_
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## Where has this been tested?
…

## Any other comments?
…

### CODE REVIEW

File: electrum/eth_wallet.py [hunk 1]
```diff
@@ -185,9 +185,13 @@ def get_total_balance(self):
...
 
     def get_all_balance(self, wallet_address, from_coin):
-        if len(self.total_balance) != 0:
-            if len(self.total_balance['balance_info']) != 0 and (time.time() - self.total_balance['time'] > 10):
-                return self.total_balance['balance_info']
+        if (
+                self.total_balance
+                and self.total_balance.get("balance_info")
+                and self.total_balance.get("time")
+                and time.time() - self.total_balance["time"] <= 10  # Only cache for 10s
```
Comment (taimanhui) [inline electrum/eth_wallet.py:192]: 确定`self.total_balance["time"]`不需要做类型转换？

**Review by taimanhui — APPROVED** [review]
这个看起来是原来的逻辑写反了导致总是不过期么？

File: electrum/eth_wallet.py (hunk 1 above)
Comment (huazhouwang) [reply electrum/eth_wallet.py:192]: 嗯，不需要做类型转换

**Comment by huazhouwang** [comment]
> 这个看起来是原来的逻辑写反了导致总是不过期么？

嗯，是的

**Review by shuaiLiWang — APPROVED** [review]

**Pull request merged**